Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM php:5.6
```

Comment:
proč ne 7.1?

Reply:
nechcel som to robit v tomto PR, chcem aby sa to co najviac podobalo sucasnemu "runtime" prostrediu

Reply:
skusil som to s 7.1 a testy presli tak to teda dam na 7.1